Kong/kuma-demo
# CVE-2020-7598 package resolutions

## PR Details

This PR addresses [this security issue](https://github.com/Kong/kuma-demo/network/alert/app/yarn.lock/minimist/open). I've added resolutions for the `braces` and `minimist` packages since they seem to be the ones affected in the context of this project.

The PR title mentions [CVE-2020-7598](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-7598) but I found similar security reports for other packages too. This covers them all. 

### How do I check?

1. Pull this PR locally
2. `cd` into `/app` and delete your `node_modules` folder
3. Run `yarn install`
4. Run `yarn audit` and you should see that there are no longer any security risks
